cypress/integration/drawing-tool.test.js

```diff
@@ -0,0 +1,51 @@
+import { phonePost, phoneListen, getAndClearLastPhoneMessage } from "../support";
+
+const authoredStateWithoutStamps = {
+  version: 1,
+  questionType: "open_response",
```

Comment:
I don't think "open_response" should be here. I think you want to use `"iframe_interactive"` (or `"iframe_interactive" as const` to make TypeScript happy).